control_node: {
  control_is_on: false,
  joint_angle_desired: [0.634375, -1.0156591796875, -0.2863896484375, 1.140431640625, 0.121525390625, 1.4903896484375, 3.6591357421875,],
  cartesian_pose_desired: [0,0,0,0,0,0],
  joint_velocity_desired: [0,0,0,0,0,0,0],
  selected_filter_number: 3,
  Kd_upper_limit: [100,100,100,60,40,20,20],
  Kd_lower_limit: [10,10,10,7,5,5,5,],
  Dd_upper_limit: [1,1,1,1,1,1,1],
  Dd_lower_limit: [1,1,1,1,1,1,1],
  Lowpass_coeff: 0.6,
  move2neutral: true,
  suppress_self_collision_detection: false,
  suppress_contact_collision_detection: false,
}

oscillation_guard: {
  window_length: 50,
  corner_frequency: 20,
  magnitude_limit: 10,
}
stiffness_manager: {
  critical_velocity: 0.2,
  offset_angle: 5,
  critical_angle: 15,
  calculation_method: pose,
}

mediapipe: {
  start: true,
}
nullspace: {
  is_locked: false,
  jointspace_pose: [0.634375, -1.0156591796875, -0.2863896484375, 1.140431640625, 0.121525390625, 1.4903896484375, 3.6591357421875,],
}
named_poses: {
  right: {
    poses: {
      neutral: [0.634375, -1.0156591796875, -0.2863896484375, 1.140431640625, 0.121525390625, 1.4903896484375, 3.6591357421875,],
    }
  }
}
trajectory_executor: {
  method: steady,
}
